dex/networks/eth/params.go

```diff
@@ -65,6 +70,26 @@ var v0Gases = &Gases{
 	RefundGas: 43000,
 }
 
+// EncodeContractData packs the contract version and the secret hash into a byte
+// slice for communicating a swap's identity.
+func EncodeContractData(contractVersion uint32, swapKey [SecretHashSize]byte) []byte {
```

Comment:
I won't object to a rename here, but this feels right to me.

Reply:
It's good. The use of `contractData` is becoming consistent.